### PR TITLE
CONTRIB/PERF: Fix perftest batch test names

### PR DESCRIPTION
**Pull request by yosefe**
# Why
Avoid duplicate test names, so when a test fails/struck it will be easy to know its parameters

### CODE REVIEW

**Comment by yosefe** [comment]
@Akshay-Venkatesh can you pls take a look?

File: contrib/ucx_perftest_config/test_types_ucp [hunk 1]
```diff
@@ -1,42 +1,48 @@
...
+#
+ucp_iov_contig_tag_lat               -t tag_lat -D iov,contig
+ucp_iov_iov_tag_lat                  -t tag_lat -D iov,iov
+ucp_contig_tag_lat                   -t tag_lat -D contig,contig
 #IOV with RNDV is not yet supported
-#ucp_contig_iov_tag_lat      -t tag_lat -D contig,iov
-ucp_iov_contig_tag_bw       -t tag_bw  -D iov,contig
-ucp_iov_iov_tag_bw          -t tag_bw  -D iov,iov
-ucp_contig_contig_tag_bw    -t tag_bw  -D contig,contig
+#ucp_contig_iov_tag_lat              -t tag_lat -D contig,iov
```
Comment (Akshay-Venkatesh) [inline contrib/ucx_perftest_config/test_types_ucp:8]: Is this test commented on purpose?

File: contrib/ucx_perftest_config/test_types_ucp [hunk 2]
```diff
@@ -1,42 +1,48 @@
...
-ucp_contig_contig_cuda_tag_bw    -t tag_bw  -D contig,contig -m host,cuda
-ucp_contig_cuda_stream_bw        -t stream_bw  -r recv_data -m cuda
-ucp_contig_cuda_stream_lat       -t stream_lat -r recv_data -m cuda
-ucp_contig_cuda_stream_bw        -t stream_bw  -r recv -m cuda
-ucp_contig_cuda_stream_lat       -t stream_lat -r recv -m cuda
-ucp_contig_contig_cuda_mng_tag_lat   -t tag_lat -D contig,contig -m cuda-managed
-ucp_contig_contig_cuda_mng_tag_bw    -t tag_bw  -D contig,contig -m cuda-managed
-ucp_contig_cuda_mng_stream_bw        -t stream_bw  -r recv_data -m cuda-managed
-ucp_contig_cuda_mng_stream_lat       -t stream_lat -r recv_data -m cuda-managed
+#ucp_contig_iov_tag_bw               -t tag_bw  -D contig,iov
```
Comment (Akshay-Venkatesh) [inline contrib/ucx_perftest_config/test_types_ucp:13]: commented on purpose?

File: contrib/ucx_perftest_config/test_types_ucp [hunk 3]
```diff
@@ -1,42 +1,48 @@
...
-ucp_contig_contig_cuda_tag_bw  -I -E sleep  -t tag_bw  -D contig,contig -m host,cuda
+#
+# CUDA wakeup mode
+#
+ucp_contig_cuda_tag_lat_sleep        -I -E sleep  -t tag_lat -D contig,contig -m cuda,cuda
+ucp_contig_cuda_host_tag_lat_sleep   -I -E sleep  -t tag_lat -D contig,contig -m cuda,host
+ucp_contig_host_cuda_tag_lat_sleep   -I -E sleep  -t tag_lat -D contig,contig -m host,cuda
+ucp_contig_cuda_tag_bw_sleep         -I -E sleep  -t tag_bw  -D contig,contig -m cuda,cuda
+ucp_contig_cuda_host_tag_bw_sleep    -I -E sleep  -t tag_bw  -D contig,contig -m cuda,host
+ucp_contig_host_cuda_tag_bw_sleep    -I -E sleep  -t tag_bw  -D contig,contig -m host,cuda
```
Comment (Akshay-Venkatesh) [inline contrib/ucx_perftest_config/test_types_ucp:48]: For some internal tests, we use arrays like src=[host,cuda,cuda-managed] and dst=[host,cuda,cuda-managed] and populate test parameters and test name over nested loops. Maybe that's more succinct?

**Review by Akshay-Venkatesh — APPROVED** [review]

File: contrib/ucx_perftest_config/test_types_ucp (hunk 1 above)
Comment (yosefe) [reply contrib/ucx_perftest_config/test_types_ucp:8]: `#IOV with RNDV is not yet supported`

File: contrib/ucx_perftest_config/test_types_ucp (hunk 2 above)
Comment (yosefe) [reply contrib/ucx_perftest_config/test_types_ucp:13]: `#IOV with RNDV is not yet supported`

File: contrib/ucx_perftest_config/test_types_ucp (hunk 3 above)
Comment (yosefe) [reply contrib/ucx_perftest_config/test_types_ucp:48]: we could have another dimension of batchfile but it would be bigger change

**Pull request merged**